src/custom/state/index.ts

```diff
@@ -27,22 +28,27 @@ const UNISWAP_REDUCERS = {
   lists
 }
 
+const reducers = {
+  ...UNISWAP_REDUCERS,
+  operator,
+  orders,
+  fee
+}
+
 const PERSISTED_KEYS: string[] = ['user', 'transactions', 'lists', 'orders', 'fee']
 
 const store = configureStore({
-  reducer: {
-    ...UNISWAP_REDUCERS,
-    operator,
-    orders,
-    fee
-  },
-  middleware: [...getDefaultMiddleware({ thunk: false }), save({ states: PERSISTED_KEYS })],
+  reducer: reducers,
+  middleware: [...getDefaultMiddleware({ thunk: false }), save({ states: PERSISTED_KEYS }), popupMiddleware],
```

Comment:
nit:  I would add some new lines to see the list of middlewares as a vertical list (easier to read)

Reply:
this change requires ignoring prettier so will ignore unless u feel strongly about it